script.js

```diff
@@ -148,6 +149,36 @@ function isValid() {
   }
 }
 
+function displaySection(sectionNumber) {
+  const booksList = document.querySelector('#awesome-books');
+  const addBooks = document.querySelector('#add-books');
+  const contact = document.querySelector('#contact');
+  const nav1 = document.querySelector('.nav1');
+  const nav2 = document.querySelector('.nav2');
+  const nav3 = document.querySelector('.nav3');
+
+  booksList.classList.remove('invisible');
+  addBooks.classList.remove('invisible');
+  contact.classList.remove('invisible');
+  nav1.classList.remove('active');
+  nav2.classList.remove('active');
+  nav3.classList.remove('active');
+
+  if (sectionNumber === 1) {
+    addBooks.classList.add('invisible');
+    contact.classList.add('invisible');
+    nav1.classList.add('active');
+  } else if (sectionNumber === 2) {
+    booksList.classList.add('invisible');
+    contact.classList.add('invisible');
+    nav2.classList.add('active');
+  } else {
+    booksList.classList.add('invisible');
+    addBooks.classList.add('invisible');
+    nav3.classList.add('active');
+  }
+}
+
 const books = grabbingBooks();
```

Comment:
- [ ] While the navigation works, I am unable to add books to the app. I think there may be an issue with the `current` variable in your app.

  Since I was unable to add books, I also could not check for book removal. Kindly ensure everything works as required
  
  ![image](https://user-images.githubusercontent.com/54848363/235990961-7a6b968d-844d-4d83-846f-c414188a3818.png)